app/serializers/auction_serializer.rb

```diff
@@ -43,6 +43,15 @@ def winning_bid
   private
 
   def find_winning_bid
-    RulesFactory.new(object).create.winning_bid
+    if auction_status.available?
+      NullBid.new
+    else
+      RulesFactory.new(object).create.winning_bid
```

Comment:
I know I approved that earlier, but I am curious about when we should have a dedicated factory class like this vs. a factory method in a class